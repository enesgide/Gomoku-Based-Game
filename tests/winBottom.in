place A19
place B19
place A18
place B18
place A17
place B17
place A16
place B16
place A15