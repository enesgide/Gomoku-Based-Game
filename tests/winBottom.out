Black wins!
A19B19A18B18A17B17A16B16A15
Thank you for playing!
